docs/CORE_DEV_GUIDE.md

```diff
@@ -0,0 +1,197 @@
+# Core Developer Guide
+
+Welcome, new core developer!  The core team appreciate the quality of
+your work, and enjoy working with you; we have therefore invited you
+to join us.  Thank you for your numerous contributions to the project
+so far.
+
+You can see a list of all the current core developers on our
+[@napari/core-devs](https://github.com/orgs/napari/teams/core-devs)
+GitHub team. You should now be on that list too.
+
+This document offers guidelines for your new role.  First and
+foremost, you should familiarize yourself with the project's
+[mission and values](MISSION_AND_VALUES.md).  When in
+doubt, always refer back there.
+
+As a core team member, you gain the responsibility of shepherding
+other contributors through the review process; here are some
+guidelines for how to do that.
+
+## All Contributors Are Treated The Same
+
+As a core developer, you gain the ability to merge or approve
+other contributors' pull requests.  Much like nuclear launch keys, it
+is a shared power: you must merge *only after* another core has
+approved the pull request, *and* after you yourself have carefully
+reviewed it.  (See `Reviewing` and especially `Merge Only Changes You
+Understand` below.) It should also be considered best practice to leave a
+reasonable (24hr) time window after approval before merge to ensure that other
+core developers have a reasonable chance to weigh in.
+
+We are also an international community, with contributors from many different time zones,
+some of whom will only contribute during their working hours, others who might only be able
+to contribute during nights and weekends. It is important to be respectful of other peoples
+schedules and working habits, even if it slows the project down slightly - we are in this
+for the long run. In the same vein you also shouldn't feel pressured to be constantly
+available or online, and users or contributors who are overly demanding and unreasonable   
```

Comment:
```suggestion
available or online, and users or contributors who are overly demanding and unreasonable
```
(I think the extra spaces are causing a premature newline)

Reply:
fixed